client/blocks/conversation-caterpillar/docs/example.jsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { ConversationCaterpillar } from 'blocks/conversation-caterpillar';
+import { posts } from 'blocks/reader-post-card/docs/fixtures';
+
+const ConversationCaterpillarExample = () => {
+	const commentsTree = {
```

Comment:
We do, for the main Conversations component. I'll consolidate those 👍 

There's a wider question here about where we should store fixtures shared across many components (we use `reader-post-card`'s fixtures in several blocks, for example).